app/assets/stylesheets/colors.scss

```diff
@@ -1,30 +1,8 @@
-$background-white: #FFFFFF;
-$background-grey: #EEEEEE;
-$background-blue: #E7F2F7;
-
-$grey: #2B2B2B;
-$light-grey: #DDDDDD;
-
-$border-grey: #DDDDDD;
-$border-dark-grey: #999999;
-
-$link-grey: #777777;
-$link-disabled-grey: #999999;
-
-$text-grey: #999999;
-$text-dark-grey: #666666;
-$text: #333333;
-
-$white: white;
-$black: black;
-$green: #8EDE3D;
-$light-green: lighten($green,20%);
-$red: #A80000;
-$blue: #3F8FBA;
-$dark-blue: darken(#0984C8,10%);
-
-$sidebars-background: #f0f0f0;
-$sidebars-sub-background: darken($sidebars-background, 12%);
-$left-navbar-drawer-background: darken($sidebars-background, 6%);
-
-$card-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
+.gray {
+  color: $text-grey;
```

Comment:
AFAIK we stick to AE in most cases. (Bootstrap variables, color vs. colour, ...) Some of our color variables still use BE but replacing those would introduce a lot of conflicts (160 occurrences of “grey”) so I think it should be part of another PR. I spoke with @jhass about a similar PR which would do some changes like replacing “stream_element” with “stream-element” and we agreed that this should happen after the 0.5.4.0 release to reduce the amount of conflicts for the backport merges.